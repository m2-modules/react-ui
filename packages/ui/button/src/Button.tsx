import React from "react";

export interface ButtonProps {
  text?: string;
}

export default function Button({ text = "Button" }: ButtonProps) {
  console.log("test");
  return <button>{text}</button>;
}
