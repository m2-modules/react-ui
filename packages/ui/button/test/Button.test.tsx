import React from "react";
import renderer from "react-test-renderer";
import Button from "../src/Button";

it("show button on document", () => {
  const component = renderer.create(<Button />);

  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
