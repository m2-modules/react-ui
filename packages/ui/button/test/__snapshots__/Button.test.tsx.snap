// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`show button on document 1`] = `
<button
  onClick={[Function]}
>
  Button
</button>
`;
